node-installer/k3s-kwasm-job.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
Can you add a comment to the top of this doc about why it differs from the other kwasm job?

Reply:
oh wait this is a mistake I accidentally checked in generated yamls. Will remove